app/javascript/lib/scp-api.js

```diff
@@ -767,31 +767,47 @@ export default async function scpApi(
     } else {
       return [response, perfTimes, true]
     }
-  } else {
-    if (response.status === 401 || response.status === 403) {
-      showMessage(
-        <div>
-          Authentication failed<br/>
-          Your session may have timed out. Please sign in again.<br/><br/>
-        </div>,
-        'api-auth-failure',
-        {
-          source: 'api',
-          url,
-          isError: true,
-          messageType: 'error-client',
-          statusCode: response.status
-        }
-      )
-    }
+  } else if (response.status === 401 || response.status === 403) {
+    showMessage(
+      <div>
+        Authentication failed<br/>
+        Your session may have timed out. Please sign in again.<br/><br/>
+      </div>,
+      'api-auth-failure',
+      {
+        source: 'api',
+        url,
+        isError: true,
+        messageType: 'error-client',
+        statusCode: response.status
```

Comment:
It'd be cool if we had an overview of the various event names and properties we use to log errors to Mixpanel.  My hunch is that we've accreted several overlapping but unnecessarily inconsistent ways to do that.

This code is pre-existing, so it's definitely not a blocker.  It just caught my attention because I've been querying `error` events recently, and sends a different set of data to our analytics.